### DISCUSSION TITLE
Configuration help on Edgemesh

### DISCUSSION THREAD

**Discussion by siredmar** [kubeedge/kubeedge#4293] [Q&A]
Hi!

I need some help on the edgemesh configuration. 
The only thing from edgemesh i need is that i have a node local DNS running, that can resolve to a service address if a matching pod is on the specific node. 

Example: I have a daemonset with some pods `myapp` with a service that resolves to `mapp-svc.myapp.svc.cluster.local`. Every Pods are running on every edge node. Another pod running on that node shall connect to the `myapp` if it conneects via `mapp-svc.myapp.svc.cluster.local`. 
I don't need node2node or edge2cloud or cloud2edge service access. Only same node. It's okay for me if there is only one relay running in the cloud. No need for edge relays. 

How is this configured correctly? 

Do i need modules.edgeDNS enabled? Do i need modules.edgeDNS.cacheDNS enabled?
Do i need modules.edgeProxy enabled?

**Comment by siredmar** [comment]
BTW, i'm testing with a kind cluster and using some iptables rules to let my RPI connect to my kind cluster.

**Comment by vincentgoat** [comment]
Hi,
I think EdgeApplication and NodeGroup features can filter endpoints of pods according to NodeGroup which can allow you to connect to other pods only at the same node.

**Reply by siredmar** [reply]
So i don't need edgemesh at all?

**Reply by vincentgoat** [reply]
edgemesh still needed, just filter endpoints by EdgeApplication and Nodegroup.

**Reply by siredmar** [reply]
So, how would i do that?

**Reply by vincentgoat** [reply]
FYI: https://github.com/kubeedge/kubeedge/blob/master/docs/proposals/node-group-management.md#example

**Reply by siredmar** [reply]
Ok, that's nice, but this brings me to my original question on the correct edgemesh configuration.
